Backend/Remora.Discord.Core/Optional.cs

```diff
@@ -60,20 +60,18 @@ namespace Remora.Discord.Core
             }
         }
 
-        // Technically, this is a lie, but since the nullability of the type is reliant on the actual generic type and
-        // its annotations, this warning can be disabled.
-        #pragma warning disable CS8775
         /// <inheritdoc />
-        [MemberNotNullWhen(true, nameof(Value))]
```

Comment:
Value is never null. It is initialized to `false` and only set high when a value is provided in the relevant constructor.